### ISSUE TITLE
Unable to access to content

### ISSUE THREAD

**Issue by csi0n** [ixudra/curl#38]
Found a problem, if the server returns the state of the data is 200 so normal access to content, but if not 200, such as the 500，400 status code will not be able to normal access to content

**Comment by elimentz** [comment]
Can you show me the request you are trying to do? Are you using the response object or not?

**Comment by csi0n** [comment]
This is my code.
```
$curl = Curl::to("https://api.center.medsci.cn/MedSciUser/User/register")
                    ->withData([
                        'username' => $request->mobile,
                        'password' => $request->password,
                        'code' => $request->checkcode
                    ])
                    ->enableDebug(storage_path('logs/curl.log'))
                    ->returnResponseObject()
                    ->post();
```

**Comment by elimentz** [comment]
Fixed in v6.8.1
